Bottom: e83a3ba5ad73d6644c39499779563379ef136fad
Top:    2e7ded39c261a98279bfd01369b130ef0dcab294
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-01-31 04:20:04 +0000

Refresh of drop-in-pcb-gl-code-various-me

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 0c4da62..20673eb 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -1071,6 +1071,7 @@ ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
 
   ghid_init_triangle_array ();
   DrawAttached (TRUE);
+  DrawMark (TRUE);
   ghid_show_crosshair (TRUE);
 
   ghid_flush_triangles ();
